helpers/inspector_files.rb

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2013-2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+
+
+# This script can be used to generate files with information
+# of each scope of an inspected system automatically.
+# This is usefull for the integration tests.
+
+# Usage: rake inspector_files[ip-adress,distribution]
+
+require "fileutils"
+require "cheetah"
+
+class CreateScopeInfoTestData
+  def initialize
+    @machinery = File.expand_path("../bin/machinery", File.dirname(__FILE__))
+    @test_data_path = File.expand_path("../spec/data/", File.dirname(__FILE__))
+  end
+
+  def inspect_system(ip_adress)
+    Cheetah.run(@machinery, "inspect", ip_adress, "-n", "machine", stdout: :capture)
```

Comment:
This writes a description named `machine` to the users machinery directory. This could cause conflicts. Maybe better use a random name and remove the description after its use.